package.json

```diff
@@ -21,6 +21,8 @@
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn constraints --fix",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!**/CHANGELOG.md' '!**/CHANGELOG.old.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore",
     "prepack": "./scripts/prepack.sh",
+    "publish-previews": "yarn workspaces foreach --parallel run publish-preview",
```

Comment:
Nit: Perhaps this should be singular to follow the naming strategy we use for other scripts that operate across the monorepo? (e.g. `build:docs`, `changelog:validate`, `test`)